Formula/timg.rb

```diff
@@ -0,0 +1,28 @@
+class Timg < Formula
+  desc "Terminal image and video viewer"
+  homepage "https://timg.sh/"
+  url "https://github.com/hzeller/timg/archive/refs/tags/v1.2.1.tar.gz"
+  sha256 "2ef2e1b79c3bf82e951ad35628bc0387fe8260052614a213385de82fcfa901ec"
+  license "GPL-2.0-only"
+  head "https://github.com/hzeller/timg.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "ffmpeg"
+  depends_on "graphicsmagick"
+  depends_on "jpeg-turbo"
+  depends_on "libexif"
+  depends_on "libpng"
+  depends_on "webp"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/timg", "--version"
+    system "#{bin}/timg", "-g10x10", test_fixtures("test.gif")
+    system "#{bin}/timg", "-g10x10", test_fixtures("test.jpg")
+    system "#{bin}/timg", "-g10x10", test_fixtures("test.png")
```

Comment:
Can we check the output of these commands? Maybe even just the first few characters, with an `assert_match`.